capstone/pom.xml

```diff
@@ -84,7 +84,14 @@
         <version>2.2.0</version>
         <configuration>
           <deploy.projectId>step90-2020</deploy.projectId>
-          <deploy.version>1</deploy.version>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>11</source>
```

Comment:
I think the build is failing because you're using Java 11 and CI build is configured to use Java 8.
Why did you change to Java 11?

Reply:
See: https://github.com/googleinterns/step90-2020/blob/master/.github/workflows/maven.yml

Reply:
I updated the CI in #65, but you probably need to sync to master to get the effect.

Reply:
Thanks for finding the problem for me. I was struggling to identify where the issue was because it was building and running fine locally. This is so odd because I never changed it to Java 11.

Reply:
I wonder how `maven-compiler-plugin` got in the pom then. Maybe IntelliJ inserted it?
In any case, in makes sense to be on 11 because we're deploying to App Engine that runs Java 11.